Resources/Private/Language/de.locallang.xlf

```diff
@@ -3,6 +3,10 @@
 	<file source-language="en" datatype="plaintext" original="messages" date="2013-03-40T10:38:40Z" product-name="events2">
 		<header/>
 		<body>
+			<trans-unit id="plugin.settings.moreLabel">
+				<source>", ...</source>
```

Comment:
You really want to translate comma, dots and quotes? :-)
In this case I would transfer these signes into the VH directly